queries/lalrpop/highlights.scm

```diff
@@ -0,0 +1,57 @@
+[
+ 	"pub"
+	"grammar"
+	"match"
+	"extern"
+	"type"
+	"enum"
+] @keyword
+
+[
+ 	"+"
+	"*"
+	"?"
+] @operator
+
+(grammar_type_params
+	"<" @punctuation.bracket
+	">" @punctuation.bracket)
+
+(symbol
+	"<" @punctuation.bracket
+	">" @punctuation.bracket)
+
+(binding_symbol
+	"<" @punctuation.bracket
+	">" @punctuation.bracket)
+
+(binding_symbol
+	name: (identifier) @variable.parameter)
```

Comment:
Please use `@parameter` to make  CI happy

Reply:
Done (force-pushed)